src/main/java/ch/admin/bag/covidcertificate/gateway/client/internal/DefaultIdentityAuthorizationClient.java

```diff
@@ -66,16 +67,23 @@ private boolean checkUserExists(QueryUsersResponse response) {
         }
     }
 
-    private boolean checkUserRoleExists(QueryUsersResponse response) {
+    protected boolean hasUserRoleSuperUserOrCreator(QueryUsersResponse response) {
```

Comment:
IMHO : response -> users or usersResponse
...would give a better idea of what it is about/context, response being too generic/general.